app/routes/account.$accountId.$appId.insights/route.tsx

```diff
@@ -26,6 +26,12 @@ export const loader: LoaderFunction = async ({ request, params }) => {
   await requireUser(request)
   const url = new URL(request.url)
   const daysParam = Number(url.searchParams.get("days") ?? "7")
+  const { accountId, appId } = params
+
+  // Prevent manually entering daysParam
+  if (daysParam !== 7 && daysParam !== 30 && daysParam !== 60) {
+    return redirect(`/account/${accountId}/${appId}/insights`)
```

Comment:
would it make sense to change the param instead of redirecting?

`url.searchParams.set("days", "7")`

Reply:
That didn't achieve the same behavior, but I use this instead  `redirect(url.pathname)`